mahta/JS/JS data fetching/new-post.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Posts App</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+      <div class="add">
+      <input type="text" id="postTitleInput" placeholder="Title" />
+      <input type="text" id="postUserIdInput" placeholder="UserID" />
```

Comment:
Rewrite the input tag and get the number from the client.

Reply:
I think it's already done in my submitPost function, but I'm waiting for further instructions if it's not what you meant.

Reply:
I can submit a fake ID with characters like 'ABC' :)

Reply:
@MahtaMirzaei